### PR TITLE
accel: T2631: Add option for radius disable-accounting

### PR DESCRIPTION
**Pull request by sever-sever**
Add new radius server option "disable-accounting" for l2tp/sstp/pptp

### CODE REVIEW

File: data/templates/accel-ppp/config_chap_secrets_radius.j2 [hunk 1]
```diff
@@ -5,7 +5,7 @@ chap-secrets={{ chap_secrets_file }}
 [radius]
 verbose=1
 {%   for server, options in authentication.radius.server.items() if not options.disable is defined %}
-server={{ server }},{{ options.key }},auth-port={{ options.port }},acct-port={{ options.acct_port }},req-limit=0,fail-time={{ options.fail_time }}
+server={{ server }},{{ options.key }},auth-port={{ options.port }},{% if options.disable_accounting is defined %}acct-port=0{% else %}acct-port={{ options.acct_port }}{% endif %},req-limit=0,fail-time={{ options.fail_time }}
```
Comment (c-po) [inline data/templates/accel-ppp/config_chap_secrets_radius.j2]: Why not adjust the accounting port to 0 as with the lagacy implementation in the `get_accel_dict()` function? This implementation adds logic to a stupid template which should be avoided.

https://github.com/vyos/vyos-1x/blob/current/python/vyos/configdict.py#L402

**Review by c-po — CHANGES_REQUESTED** [review]

File: data/templates/accel-ppp/config_chap_secrets_radius.j2 (hunk 1 above)
Comment (sever-sever) [reply data/templates/accel-ppp/config_chap_secrets_radius.j2]: I'm afraid I don't understand how it works. And what do I need to do.

Comment (c-po) [reply data/templates/accel-ppp/config_chap_secrets_radius.j2]: I'd extend the common `get_accel_dict()` functionality which can and should be used by any Accel-PPP service in the same way you did for `src/conf_mode/vpn_l2tp.py`. Thus it's coded one time on value retrieval and no hacks in templates are required. Chat me up on Slack if you need more infos.

**Comment by c-po** [comment]
Please also extend the `test_authentication_radius()` testcase here https://github.com/vyos/vyos-1x/blob/current/smoketest/scripts/cli/base_accel_ppp_test.py#L137

File: data/templates/accel-ppp/config_chap_secrets_radius.j2 (hunk 1 above)
Comment (sever-sever) [reply data/templates/accel-ppp/config_chap_secrets_radius.j2]: I have updated the PR and am not changing the template for sstp "config_chap_secrets_radius.j2"

**Pull request merged**